src/createShots.ts

```diff
@@ -30,34 +31,25 @@ export const createShots = async () => {
   if (ladleShots) {
     const { ladleUrl } = ladleShots;
 
-    const {
-      data: ladleMeta,
-    }: {
-      data: {
-        stories: {
-          id: string;
-        };
-      };
-    } = await axios.get(`${ladleUrl}/meta.json`);
-    const collection: Story[] | undefined = Object.keys(ladleMeta.stories).map(
-      (storyKey) => ({ id: storyKey, story: storyKey, kind: storyKey }),
-    );
-
-    if (!collection || collection.length === 0) {
-      throw new Error('Error: Stories not found');
-    }
+    try {
+      const collection = await collectLadleStories(ladleUrl);
 
-    log(`Found ${collection.length} ladle stories`);
+      if (!collection || collection.length === 0) {
+        throw new Error('Error: Stories not found');
+      }
 
-    ladleShotItems = generateLadleShotItems(ladleUrl, collection);
+      log(`Found ${collection.length} ladle stories`);
 
-    log(`Prepared ${ladleShotItems.length} ladle stories for screenshots`);
+      ladleShotItems = generateLadleShotItems(ladleUrl, collection);
 
-    log({ ladleShotItems });
+      log(`Prepared ${ladleShotItems.length} ladle stories for screenshots`);
 
-    await takeScreenShots(ladleShotItems);
+      await takeScreenShots(ladleShotItems);
 
-    log('Screenshots done!');
+      log('Screenshots done!');
+    } catch (error: unknown) {
+      throw error;
```

Comment:
If we are not doing anything with the caught error (e.g. log it) we might also just skip the try-catch block altogether.